ansible.cfg

```diff
@@ -0,0 +1,6 @@
+[defaults]
+ask_sudo_pass = True
+transport = ssh
+
+[ssh_connection]
+ssh_args = -o ForwardAgent=yes
```

Comment:
but it's working without the config file if I use the --ask-sudo-pass parameter again.